src/openrct2/ride/Vehicle.h

```diff
@@ -109,6 +109,7 @@ struct Vehicle : EntityBase
     int32_t acceleration;
     RideId ride;
     uint8_t vehicle_type;
+    bool isReversed;
```

Comment:
PascalCase for class members

Reply:
Is there no room in the flags field(s) for one more parameter? It seems like a waste of 7 bits to put this on its lonesome.

Reply:
I did some testing and it works as an update flag, this also eliminates the need for a new save format version. Would the flag be modifiable via plugin using this implementation?

Reply:
A getter/setter for this flag would be fine.